src/components/Progress.js

```diff
@@ -0,0 +1,24 @@
+import { PropTypes } from 'prop-types';
+import { React } from 'react';
+
+function Progress(props) {
+  const { progress } = props;
+  return (
+    <div className="progress-container">
+      <div className="progress" style={{ backgroundImage: `conic-gradient(#0290ff ${progress}%, whitesmoke ${progress}%)` }}>
```

Comment:
- [ ] [OPTIONAL] For best practices kindly don't use inline styles, if you want to you can use a function to group the styles and apply them to the component/tag
e.g.
```
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){

    return <div style={myComponentStyle}> Inline Styled Component</div>

}
```

Reply:
Thank you @Ismailco